download.tt

```diff
@@ -139,7 +139,7 @@
         <aside class="notice-box -warning">
           <strong>This is a demo appliance</strong> for VirtualBox (in OVA format).
         </aside>
-        <p>VirtualBox image includes includes <strong>KDE (Plasma5)</strong> as
+        <p>VirtualBox image includes the <strong>Plasma Desktop</strong> as
```

Comment:
```suggestion
        <p>VirtualBox image includes the <strong>Plasma Desktop</strong> (was KDE) as
```
I think a lot of linux users are unaware that Plasma Desktop is what KDE used to be.